data/migrations/25.lua

```diff
@@ -1,3 +1,16 @@
 function onUpdateDatabase()
-	return false
+	print("> Updating database to version 25 (Store inbox changes)")
+	db.query([[
+		CREATE TABLE IF NOT EXISTS `player_storeinboxitems` (
+		  `player_id` int(11) NOT NULL,
+		  `sid` int(11) NOT NULL,
+		  `pid` int(11) NOT NULL DEFAULT '0',
+		  `itemtype` smallint(6) NOT NULL,
+		  `count` smallint(5) NOT NULL DEFAULT '0',
+		  `attributes` blob NOT NULL,
+		  UNIQUE KEY `player_id_2` (`player_id`, `sid`),
+		  FOREIGN KEY (`player_id`) REFERENCES `players`(`id`) ON DELETE CASCADE
+		) ENGINE=InnoDB DEFAULT CHARACTER SET=utf8;
```

Comment:
Shouldn't there be foreign key on `parent_id`?

Reply:
Foreign key to what? pid - parent_id is id of another container in the same table, this table is cloned inboxitems table and few others

Reply:
I can change these, but then it will be inconsistent with rest of the tables that got the same structure like player_depotitems, player_inboxitems etc.

Reply:
Foreign key to the same table. You can make parent_id to have a foreign key constraint on `id`, although it would have to default to NULL, as 0 is likely not a valid record.

I believe we were talking about changing all the other tables as well, but it was tabled until 2.0 because it's a change that could be breaking. Here, you're creating a new table, so it's easy to make the column names sound from the get-go.

Reply:
Kinda don't get it also don't get it why would we need that.
Did you mean?
```
FOREIGN KEY (`parent_id`) REFERENCES `player_storeinboxitems`(`item_type`) ON DELETE CASCADE
```